registry-library/library/library_test.go

```diff
@@ -981,3 +981,151 @@ func TestDownloadStarterProjectAsDir(t *testing.T) {
 		})
 	}
 }
+
+func TestPullStackFromRegistry(t *testing.T) {
+
+	registry := "https://registry.stage.devfile.io"
```

Comment:
We need a registry with an oci server to test with because we need to verify if the [manifest file ](https://github.com/opencontainers/image-spec/blob/main/manifest.md#image-manifest) is set up correctly.   With issue https://github.com/devfile/api/issues/1089, the manifest did not contain the custom devfile [config media type](https://github.com/devfile/registry-support/blob/main/index/server/pkg/server/constants.go#L31) and it did not contain all the descriptors associated with the supported media types.  Because the devfile was pushed to the oci registry with a bad manifest, not all resources could be pulled.

Reply:
> We need a registry with an oci server to test with because we need to verify if the [manifest file](https://github.com/opencontainers/image-spec/blob/main/manifest.md#image-manifest) is set up correctly.

In this case, could this variable be changed to use an environment variable with the default value being `https://registry.stage.devfile.io`? Just in case we want (or have to) to target a different registry deployment. Might be good to make this a global variable too, for all test cases which needs an actual registry server.